pkg/mod/load.go

```diff
@@ -0,0 +1,93 @@
+package mod
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func LoadAll() error {
```

Comment:
a func which takes no arguments and is not a method receiver means global variables, which is never safe. I'm sure we can come up with a version of this without the need for globals.